README.md

```diff
@@ -14,9 +15,13 @@ NOTE: This is also maintained at openshift/aos-cd-jobs repo [https://github.com/
 - scale-ci-watcher
 - scale-ci-linter
 
+![Alt text](images/scale-ci-workflow.png)
+
+
 ### Properties files
 The parameters for each job in the scale-ci-pipeline are supplied through a job specific properties file. It contains key=value pairs, sample properties for all the supported jobs are hosted in scale-ci-pipeline/properties-files.
 
+The properties for the OCP large scale and per sprint test runs are maintained at <https://github.com/redhat-performance/scale-ci-properties>.
```

Comment:
Same as above, just make a link with the org/repo-name. I don't think there's any appeal to having the full URL spelled out.